hack/tools/go.mod

```diff
@@ -7,12 +7,10 @@ require (
 	github.com/joelanford/go-apidiff v0.1.0
 	github.com/onsi/ginkgo v1.16.5
 	github.com/vmware/govmomi v0.23.1
-	k8s.io/code-generator v0.22.2
-	sigs.k8s.io/cluster-api/hack/tools v0.0.0-20211104153216-fb1f86267fed
+	k8s.io/code-generator v0.27.2
```

Comment:
Shouldn't matter that those got bumped in hack/tools as a consequence of bumping cluster-api/hack/tools.

I bumped cluster-api/hack/tools as the newer version of the release note tool is better